### PR TITLE
Update README documentation and deprecate old docs

### PR DESCRIPTION
**Pull request by monkecheese**

### CODE REVIEW

**Comment by monkecheese** [comment]
Thank you @levionessa.  I've switched to semantic linefeed style, and wrote out some docs for installing, running tests, deploying, and added badges for our CI/CD pipeline.  After I get a couple 👍 , I feel ready to merge this (or iterate further if there's anything glaring that needs to be changed) .  😄

**Comment by levionessa** [comment]
Yeah I think this is great it you should :shipit:. Bonus points with the semantic line feeds is Github parses single line line feeds into paragraphs so it looks like this: https://github.com/lumenlearning/OpenAssessments/blob/8c28294e716cb94c046b713c5ffd667341d5dfc7/README.md

File: README.md [hunk 1]
```diff
@@ -1,316 +1,186 @@
...
+and
+[nvm](https://github.com/creationix/nvm)
+for Ruby and Node version management, respectively.
 
-http://www.openassessments.com/assessments/load?confidence_levels=true&eid=atest&src_url=https%253A%252F%252Fdl.dropboxusercontent.com%252Fu%252F7594429%252FedXCourse%252Fsequential%252F97cc2d1812204294b5fcbb91a1157368.xml
+* Ruby (v2.2.2)
+* Rails
+* Bundler
+* Node (v0.10.37)
+* Postgresql
```
Comment (reedlauber) [inline README.md]: Maybe a reference to the minimum Postgres version here?

Comment (monkecheese) [reply README.md]: Oh yes, that'd be nice.  Good catch!

**Comment by levionessa** [comment]
Yeah I think this is great it you should :shipit:. Bonus points with the semantic line feeds is Github parses single line line feeds into paragraphs so it looks like this: https://github.com/lumenlearning/OpenAssessments/blob/8c28294e716cb94c046b713c5ffd667341d5dfc7/README.md

**Pull request merged**